sos/report/plugins/instructlab.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2024 Red Hat, Inc., Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, IndependentPlugin, PluginOpt)
+
+
+class Instructlab(Plugin, IndependentPlugin):
+
+    short_desc = 'Instructlab'
+    plugin_name = 'instructlab'
+    profiles = ('ai',)
```

Comment:
Not really happy about this, but 'artificial-intelligence' and similar names were too long. I can remove the profile name for now if there are no other suggestions and this one doesn't look good.

Reply:
I would be more concise here. https://instructal.ai advertises itself as "A new community-based approach to build truly open-source LLMs" and that seems legit for me (well, haveng the "a new (thing)" in sos source for years will become ridiculous, but.. at least not now).

Reply:
So you want me to remove everything and add the instructlab.ai definition?

Reply:
Ah sorry @pmoravec I meant the profile, 'ai'. If feels kind of weird that's so small, but I don't think there's another way to specify it. And yes, there are more plugins that should go into this profile, but I'm still waiting for the whole list. 
For the doctext I tried to copy what we have in our docs, because I thought it would be clearer, and then added a note that this can run both as container and as a non-container. Should I try to make it clearer?